core/templates/pages/topics-and-skills-dashboard-page/modals/create-new-topic-modal.component.ts

```diff
@@ -44,6 +45,8 @@ export class CreateNewTopicModalComponent extends ConfirmOrCancelModal {
 
   topicUrlFragmentExists: boolean = false;
   topicNameExists: boolean = false;
+  max_web_title_frag = constants.MAX_CHARS_IN_PAGE_TITLE_FRAGMENT_FOR_WEB;
+  min_web_title_frag = constants.MIN_CHARS_IN_PAGE_TITLE_FRAGMENT_FOR_WEB;
```

Comment:
Hm, do we use snakecase in the frontend for variable names in other places?

Reply:
We generally use camel-case for naming variables in the frontend. I have now fixed this mistake.